OpenRA.Game/Game.cs

```diff
@@ -59,13 +59,13 @@ public static class Game
 
 		static Task discoverNat;
 
-		public static OrderManager JoinServer(string host, int port, string password, bool recordReplay = true)
+		public static OrderManager JoinServer(string host, int port, string password, bool recordReplay = true, bool isSkirmish = false)
```

Comment:
I really don't like the idea of adding this, but unfortunately don't have the time to think about this and offer an alternative.